packages/insomnia-app/app/ui/components/codemirror/code-editor.js

```diff
@@ -312,7 +312,11 @@ class CodeEditor extends React.Component {
         let startToken = '{';
         let endToken = '}';
 
+        // Prevent retrieving an invalid content if undefined
+        if (!from.line || !to.line) return '\u2194';
```

Comment:
Unfortunately, if `from` or `to` are undefined, this will throw an error. This will need to be changed to:

```suggestion
        // Prevent retrieving an invalid content if undefined
        if (!from?.line || !to?.line) return '\u2194';
```